src/core/blocks/core.hpp

```diff
@@ -1543,6 +1544,9 @@ struct Sequence : public SeqBase {
 
   void addType(Types &inner, BasicTypes type) {
     switch (type) {
+    case BasicTypes::None:
```

Comment:
In which cases does this happen?

Reply:
None is a value, you could have a sequence on nones :)
Also we need to cover all cases anyway.